RT.jenkinsfile

```diff
@@ -123,13 +125,14 @@ pipeline {
 			}
 
 			steps {
-				build/norma run --label b1 -o . release_testing/b1.NewValMidRun.yml
+				build/norma run --label b1 -o . scenarios/release_testing/b1.NewValMidRun.yml
 			}
 
 			post {
 				success {
-					dir('./norma_data_b1_latest') {
-						archiveArtifacts artifacts: '**'
+					script {
```

Comment:
post->success will be changed to post->always.

Do note that if a norma run failed, it may fail in such a way that the retrieval of logs does not work. (i.e. docker container crahsed unexpectedly, so we cannot exec a command to retrieve the logs directly to it). In such cases, the retrieved logs might not be complete.